runtime/src/launch/aprun/aprun-utils.c

```diff
@@ -211,10 +230,11 @@ static char _Nbuf[16];
 static char _jbuf[16];
 char** chpl_create_aprun_cmd(int argc, char* argv[],
                              int32_t numLocales, const char* _ccArg) {
-  char *largv[8];
+  char *largv[16];
```

Comment:
Why not 9?  That's all we need.  Making it larger (against future needs?) seems like it will increase the likelihood someone will overrun it because of not realizing they need to expand it.  (Ideally we'd add limit checking, but see "weep" above ...)

Reply:
I'll size this to fit.  Sadly, the function would need to be rearchitected for a limit check to even work, since the way it is now, a limit check would have to be at the end, after the undefined behavior has already occurred.